Engine/platform/osx/acplmac.cpp

```diff
@@ -28,6 +28,7 @@ using namespace AGS::Common;
 
 void AGSMacInitPaths(char appdata[PATH_MAX]);
 void AGSMacGetBundleDir(char gamepath[PATH_MAX]);
+int AGSMacGetFreeSpaceInMB(const char path[PATH_MAX]);
```

Comment:
Argument should not be `const char path[PATH_MAX]`, but `const char*`, because there's no guarantee it's a array of PATH_MAX size.